src/Compatibility/ControlGallery/src/Issues.Shared/Bugzilla47923.cs

```diff
@@ -129,7 +129,6 @@ public LandingPage()
 		}
 
 #if UITEST
-[Microsoft.Maui.Controls.Compatibility.UITests.FailsOnMauiAndroid]
 [Microsoft.Maui.Controls.Compatibility.UITests.FailsOnMauiIOS]
 		[Test]
 		public void Bugzilla47923Test()
```

Comment:
I have re-launched the UITests part of the build, but on the first pass this test failed on Android.